integration_tests/jpms_tests/src/main/java/org/apache/fury/integration_tests/Test.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.fury.integration_tests;
+
+import org.apache.fury.Fury;
+import org.apache.fury.benchmark.Benchmark;
+import org.apache.fury.format.encoder.Encoders;
+import org.apache.fury.test.bean.Foo;
+
+/**
+ * A test class that simply references classes from the various Fury artifacts to check whether
+ * they specify the module names referenced in the module-info descriptor.
+ */
+public class Test {
```

Comment:
Could We add some testng unit test?

Reply:
What should the unit test look like? To my understanding, this integration test should only check that the module names are configured correctly and not that the code does anything reasonable (this is what the unit tests of the actual core, format, etc. modules are for). This already happens at the compile level, because if the compiler cannot find the modules, it can't link against the referenced classes  (you can check this by using an older `maven-compiler-plugin` version (e.g., `3.3.0`) which does not put the dependencies on the module-path correctly).

I could move this class to `src/test/java` but it still wouldn't really be a `@Test` class. It would only move the potential error detection from the `default-compile` execution to the `default-testCompile` execution.

Reply:
I see, thanks for detailed explanation